internal/registry/registry.go

```diff
@@ -89,29 +90,10 @@ func (r *registry) GetLayerByDigest(digest string, pullConfig *RepoPullConfig) (
 	return crane.PullLayer(pullConfig.repo+"@"+digest, pullConfig.authOptions...)
 }
 
-func (r *registry) ExtractToolkitRelease(layer v1.Layer) (*DriverToolkitEntry, error) {
```

Comment:
How are you going to check that the correct DTK was used in preflight without it?

Reply:
This is upstream code, we are not going to need it, since we moved from using json file, to check the module path